packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/src/rest/outputModels.ts

```diff
@@ -28,13 +28,13 @@ export interface OperationOutput {
 /** Localized display information for and operation. */
 export interface OperationDisplayOutput {
   /** The localized friendly form of the resource provider name, e.g. "Microsoft Monitoring Insights" or "Microsoft Compute". */
-  provider?: string;
+  readonly provider?: string;
```

Comment:
this is because of https://github.com/Azure/typespec-azure/pull/1156/files#diff-823f10c1812ffa803f58da35c7d41623a79c2e5c37f0c15d09ce66bcfdcfbe97